lib/galaxy/datatypes/converters/archive_to_directory.xml

```diff
@@ -22,8 +23,8 @@
             <option value="directory">directory</option>
         </param>
     </inputs>
-    <outputs provided_metadata_file="metadata_json">
-        <data format="directory" name="output1"/>
+    <outputs>
+        <data format="auto" name="output1"/>
```

Comment:
The previous version is a lot easier to reason about in the workflow editor and when queuing up jobs, what was wrong with it ?